pages/guides/sandbox/customers.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: guides
+title: "Customers"
+guide:
+  step: 1
+meta:
+  title: "Testing Different API Customer Types | Dwolla API Documentation"
+  description: Test and simulate adding Customers and handling verification statuses in the Sandbox before working with real end users.
+---
+
+import imageFail from "../../../assets/images/content-images/test-document-upload-fail.png";
+import imageSuccess from "../../../assets/images/content-images/test-document-upload-success.png";
+import conceptsIcon from "../../../assets/images/content-images/content-icons/concepts-icon.svg";
+import guidesIcon from "../../../assets/images/content-images/content-icons/guides-icon.svg";
+
+# Testing Customers
+
+#### Manage Customers
+
+The [Sandbox Dashboard](https://dashboard-sandbox.dwolla.com) allows you to manage Customers, as well as transfers associated with the Customers that belong to your Sandbox account. Once your application has [created its Customers](https://developers.dwolla.com/api-reference/customers/create), you can access the [Sandbox Dashboard](https://dashboard-sandbox.dwolla.com) to validate that the request was recorded properly in our test environment.
+
+<InlineCTA
+  icon={conceptsIcon}
+  text="There are multiple Customer types within the Dwolla API. Use our concept article for a more in-depth overview of each Customer type and its capabilities."
+  href="/concepts/customer-types"
+/>
+
+##### **Verified Customers**
+
+## Simulate identity verification statuses
```

Comment:
Change to h3 for aesthetics

Reply:
Resolved! - [767d461](https://github.com/Dwolla/dev-portal/pull/433/commits/767d461dc4bb4844e0e90e2ec78ef3cb2fd8dd8b)